### PR TITLE
Merge updates from the hhvm wiki INI section

### LINKED ISSUE #123: Re-merge ini wiki into the docs
I heard there were some updates to the [HHVM ini wiki](https://github.com/facebook/hhvm/wiki/INI-Settings) via @jazzdan.

We need to keep that wiki in sync with our table in these docs until we launch the docs and make that the source of truth and redirect the wiki to it.

### PR DESCRIPTION
**Pull request by jazzdan**
Fixes #123

### CODE REVIEW

File: guides/hhvm/04-configuration/02-INI-settings.md [hunk 1]
```diff
@@ -408,30 +409,30 @@ The vast majority of users will want to just set `hhvm.php7.all = 1` to fully en
 | hhvm.simple_xml.empty_namespace_matches_all | | |
 | hhvm.simulate_arm | | |
 | hhvm.small_step | | |
-| hhvm.source_root | | |
+| hhvm.source_root | For server mode, this will hold the path to the root of the directory of the code being served up. This setting is _useless_ in repo-authoritative mode. | working directory of HHVM process |
 | hhvm.spin_on_crash | | |
 | hhvm.stack_args | | |
-| hhvm.static_file.extensions[ ] | Map of filename extensions to content types for use by the Proxygen server | see [runtime-option.cpp](https://gist.github.com/JoelMarcey/29601dc033af31390fc6) for defaults |
```
Comment (jazzdan) [inline guides/hhvm/04-configuration/02-INI-settings.md]: Ah whoops, need to change this back.

Comment (JoelMarcey) [inline guides/hhvm/04-configuration/02-INI-settings.md]: Thanks for this merge. Are you making one more update with this fix before I pull?

Comment (jazzdan) [inline guides/hhvm/04-configuration/02-INI-settings.md]: Yup! Update done.

**Comment by JoelMarcey** [comment]
:+1:

**Pull request merged**